modules-js/react-fleet/src/form-elements/SelectDropdown.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import SelectDropdown from './SelectDropdown';
+
+storiesOf('Form Elements/Select', module)
+  .add('Default Dropdown', () => (
+    <React.Fragment>
```

Comment:
You can also use the shorthand `<>` instead of `<React.Fragment>`

Reply:
I generally opt to be explicit, but I'm happy to use the shorthand in our codebase.